cob_twist_controller/include/cob_twist_controller/inverse_differential_kinematics_solver.h

```diff
@@ -98,7 +101,8 @@ class InverseDifferentialKinematicsSolver
     TwistControllerParams params_;
     LimiterParams limiter_params_;
     CallbackDataMediator& callback_data_mediator_;
-    boost::shared_ptr<LimiterContainer> limiters_;
+    boost::shared_ptr<LimiterJointContainer> limiters_;
+    boost::shared_ptr<LimiterCartesianContainer> cartesian_limiters_;
```

Comment:
I'd go for naming the LimiterJointContainer `output_limiters_`
and the LimiterCartesianContainer `input_limiters_`